profile/rwahs.xml

```diff
@@ -7593,24 +7593,110 @@
       <description>Museum Users</description>
       <actions>
       </actions>
+      <typeLevelAccessControl>
+        <permission table="ca_objects" type="artwork" access="edit"/>
+        <permission table="ca_objects" type="costume" access="edit"/>
+        <permission table="ca_objects" type="museumArtefact" access="edit"/>
+        <permission table="ca_entities" type="family" access="edit"/>
```

Comment:
Should we split this out, so that there is a separate role for access to edit things like entities, storage locations, etc.  That way, it's possible to give a user access to edit objects but not, say, storage locations.